incubator/cassandra/values.yaml

```diff
@@ -109,3 +109,47 @@ securityContext:
 ## Affinity for pod assignment
 ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
 affinity: {}
+
+## Backup cronjob configuration
+## Ref: https://github.com/maorfr/cain
+backup:
+  enabled: false
+
+  # Schedule to run jobs. Must be in cron time format
+  # Ref: https://crontab.guru/
+  schedule:
+  - keyspace: keyspace1
+    cron: "0 7 * * *"
+  - keyspace: keyspace2
+    cron: "30 7 * * *"
+
+  history:
+    successfulJobs: 1
+    failedJobs: 3
+
+  annotations:
+    # Example for authorization using kube2iam
+    # Can also be done using environment variables
+    iam.amazonaws.com/role: cain
+
+  image:
+    repos: maorfr/cain
+    tag: latest
+
+  env:
+  - name: AWS_REGION
```

Comment:
Are these required fields if so comment as such otherwise comment out to take a default